### PR TITLE
chore: add runtimePlatform to fargate template

### PR DESCRIPTION
**Pull request by gsanchezgavier**
Adding the configuration of the runtimePlatform in order to facilitate the installation on graviton by changing the cpuArch to `ARM64`

### CODE REVIEW

File: deploy/fargate_sidecar_example.json [hunk 1]
```diff
@@ -2,6 +2,9 @@
     "executionRoleArn": "<YOUR_TASK_EXECUTION_ROLE>",
     "networkMode": "awsvpc",
     "requiresCompatibilities": ["FARGATE"],
+    "runtimePlatform": {
+      "cpuArchitecture": "X86_64"
```
Comment (paologallinaharbur) [inline deploy/fargate_sidecar_example.json:6]: would make sense to add also the arm one commented out with some notes?

Comment (gsanchezgavier) [reply deploy/fargate_sidecar_example.json:6]: yes but is json :sad

Comment (gsanchezgavier) [reply deploy/fargate_sidecar_example.json:6]: https://github.com/newrelic/docs-website/pull/4873

Comment (gsanchezgavier) [reply deploy/fargate_sidecar_example.json:6]: remeber that this is an example task template, I would expect that the original task with the service will already have this configuration in place.

**Review by paologallinaharbur — APPROVED** [review]

**Pull request merged**